authors/apps/articles/renderers.py

```diff
@@ -31,29 +31,26 @@ def render(self, data, media_type=None, renderer_context=None):
                 'comments': data,
                 'comments_count': len(data)
             })
-        return json.dumps({
```

Comment:
Hey, why did you remove this return in the comment renderer, won't it have effects on comments views?
Just wondering

Reply:
It is still there...just view the file